### PR TITLE
many typo fixes in the code thanks to codespell

### LINKED ISSUE #15729: Codespell report for "Matomo" (on fossies.org)
The FOSS server [fossies.org](https://fossies.org) - also supporting "Matomo" - offers a feature "[Source code misspelling reports](https://fossies.org/features.html#codespell)". Such reports are normally only generated on request, but as Fossies administrator I have just created for testing purposes such a codespell based analysis for the new Matomo release 3.13.4:

&nbsp;&nbsp;https://fossies.org/linux/www/matomo-3.13.4.tar.gz/codespell.html

Although after a first review some obviously wrong matches ("False Positives" = FPs) are already filtered out (ignored) please inform me if you find more of them so that I can force a new improved check if applicable.

Just for information there are also two supplemental pages showing some used "codespell" [configuration details](https://fossies.org/linux/www/matomo-3.13.4.tar.gz/codespell_conf.html) and all obvious [False Positives](https://fossies.org/linux/www/matomo-3.13.4.tar.gz/codespell_fps.html).

If appropriate, additional reports for the current release or also master versions (these in a special "test" folder) could be willingly created.

OT: As Fossies admin I am somewhat annoyed that a single file ("How to install Matomo.html") is located outside of the main subdirectory "matomo", as Linux user that the filename contains spaces and as a general user that the file seems superfluous since already existing within the directory "matomo/misc".

### PR DESCRIPTION
**Pull request by Findus23**
fixes #15729

thanks to @jschleus 

I'll add comments to the changes that might affect something

### CODE REVIEW

File: js/piwik.js [hunk 1]
```diff
@@ -2300,7 +2300,7 @@ if (typeof window.Piwik !== 'object') {
                 // detect this 100% correct for an iframe so whenever Piwik is loaded inside an iframe we presume
                 // the window had focus at least once.
                 hadWindowFocusAtLeastOnce = isInsideAnIframe(),
-                timeWindowLastFocussed = null,
```
Comment (Findus23) [inline js/piwik.js]: As this variable is only used locally and all uses are replaced, this should be fine (but the js files have to be regenerated)

File: plugins/SegmentEditor/lang/en.json [hunk 2]
```diff
@@ -47,7 +47,7 @@
         "UnprocessedSegmentApiError3": "Then after a few hours your segment data should become available through the API. (If it does not, there may be a problem.)",
         "CustomUnprocessedSegmentApiError1": "The Segment you requested has not yet been created in the Segment Editor and so the report data has not been pre-processed.",
         "CustomUnprocessedSegmentApiError2": "To see data for this segment, you must go to Matomo and create this segment manually in the Segment Editor.",
-        "CustomUnprocessedSegmentApiError3": "(Alternatively, you can create a new segment programatically using the SegmentEditor.add API method).",
```
Comment (Findus23) [inline plugins/SegmentEditor/lang/en.json]: invalidates translations

File: plugins/UsersManager/tests/Integration/UserAccessFilterTest.php [hunk 3]
```diff
@@ -73,7 +73,7 @@ public function setUp(): void
 
     public function test_filterUser_WithSuperUserAccess_ShouldAlwaysReturnTrue()
     {
-        $this->configureAcccessForLogin('login1');
```
Comment (Findus23) [inline plugins/UsersManager/tests/Integration/UserAccessFilterTest.php]: should not change anything

File: tests/PHPUnit/Integration/Archive/ChunksTest.php [hunk 4]
```diff
@@ -125,7 +125,7 @@ private function generateDataTableWithManySubtables($numSubtables)
         return $dataTable->getSerialized();
     }
 
-    private function createArchiveProcessorParamaters()
+    private function createArchiveProcessorParameters()
```
Comment (Findus23) [inline tests/PHPUnit/Integration/Archive/ChunksTest.php]: should not change anything

File: tests/PHPUnit/Integration/SequenceTest.php [hunk 5]
```diff
@@ -18,7 +18,7 @@
  */
 class SequenceTest extends IntegrationTestCase
 {
-    public function test_create_shouldAddNewSequenceWithInitalId1()
+    public function test_create_shouldAddNewSequenceWithInitialId1()
```
Comment (Findus23) [inline tests/PHPUnit/Integration/SequenceTest.php]: should not change anything

**Comment by sgiehl** [comment]
build js

**Review by sgiehl — APPROVED** [review]

**Comment by sgiehl** [comment]
ui files will need an update before merging

**Pull request merged**